### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28654]
#### VFS team name
Claims & Appeals

#### VFS product name
Higher Level Review

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
"Request a higher level review online" is styled as a green button when it should be styled as a link.

### Link, screenshot or steps to recreate
[Reference this previous ticket on buttons vs. links](https://github.com/department-of-veterans-affairs/va.gov-team/issues/28343) for more details

### Supporting Guidance
[04.07.1](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
[WCAG SC 3.2.4_AA](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification)

---

### Platform Recommendation
[Consider using an action link](https://design.va.gov/experimental-design/action_links) instead of styling this as a button. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by Mottie** [comment]
Fix has been merged and should be available in staging soon.

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process

**Comment by Mottie** [comment]
Work completed
